core/src/syn/v2/parser/basic.rs

```diff
@@ -140,7 +140,32 @@ impl TokenValue for f32 {
 				})?;
 				Ok(number)
 			}
-			x => unexpected!(parser, x, "an floating point"),
+			x => unexpected!(parser, x, "an floating point number"),
```

Comment:
Shouldn't this (and below too) be `a floating point number`? instead of `an`

Reply:
It should indeed, changing it.